docs/user-guide/observability.md

```diff
@@ -0,0 +1,83 @@
+
+## Integrating with the Elastic stack
+
+In this section we describe an example setup that may help you to get started if you decide to use the Elastic tool stack for the engine observability. We will assume that you have set up the Elastic stack already, and that you now want to integrate the engine components with ELK. If you do not have the Elastic stack set up yet, refer to [Elastic's documentation](https://www.elastic.co/guide/index.html) and set it up before you continue with this example.
+
+To integrate the starter kit into your ELK stack, we rely on [Metricbeat](https://www.elastic.co/beats/metricbeat) to scrape metrics from the Prometheus endpoints of the  IoT Edge modules (Network Server, Edge Hub and Edge Agent). Since Metricbeat requires access to the metric endpoints on IoT Edge, we will deploy it as an IoT Edge module, too.
+
+> Note: To [run Metricbeat on Docker](https://www.elastic.co/guide/en/beats/metricbeat/7.16/running-on-docker.html) you will need to make sure that you run IoT Edge on a supported OS/Architecture. At the time of writing, the Metricbeat Docker image does not work on ARM.
+
+To ensure that the Metricbeat is aware of all metrics endpoints that it needs to scrape and the ELK backend for the export, we need to first configure it correctly. Typically, to [configure Metricbeat on Docker](https://www.elastic.co/guide/en/beats/metricbeat/7.16/running-on-docker.html#_configure_metricbeat_on_docker), we rely on a configuration file, which we will call `metricbeat.yml` in the following. Before we discuss how to make this file available inside the Docker container, we discuss how we need to change `metricbeat.yml` to support IoT Edge metrics collection.
+
+We rely on the [Prometheus module](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-prometheus.html) to ship metrics from the Prometheus endpoints on the IoT Edge to the ELK stack. Make sure that you add the Prometheus module configuration to your `metricbeat.yml`. A basic configuration snippet might look like this:
+
+```yaml
+...
+metricbeat.modules:
+- module: prometheus
+  period: 10s
+  metricsets: ["collector"]
+  hosts: ["LoRaWanNetworkSrvModule:5000", "edgeHub:9600", "edgeAgent:9600"]
+  metrics_path: /metrics
+...
+```
+
+> Note: If you configure the Network Server to use HTTPS, it will listen on port 5001 instead of 5000. If you use self-signed certificates for this, make sure that you configure the Prometheus module with your chain of trust.
```

Comment:
```suggestion
!!! note
    If you configure the Network Server to use HTTPS, it will listen on port 5001 instead of 5000. If you use self-signed certificates for this, make sure that you configure the Prometheus module with your chain of trust.
```